rudder-lang/src/bin/main.rs

```diff
@@ -48,32 +67,60 @@ struct Opt {
     #[structopt(long, short)]
     input: PathBuf,
     /// Set to use technique translation mode
-    #[structopt(long)]
+    #[structopt(long, short)]
     translate: bool,
     /// Set to compile a single technique
-    #[structopt(long)]
-    technique: bool,
+    #[structopt(long, short)]
+    compile: bool,
     /// Output format to use
     #[structopt(long, short = "f")]
-    output_format: Option<String>,
+    output_fmt: Option<String>,
+    /// Set to change default env logger behavior (INFO, DEBUG, ERROR)
+    #[structopt(long, short)]
+    log_level: Option<LevelFilter>,
```

Comment:
Couldn't the default value be handled by structuopt instead of using an option?

Reply:
https://docs.rs/structopt/0.3.8/structopt/#default-values

Reply:
Apparently implementing Default and Display is enough to handle default values properly.

Reply:
I might be wrong but I tried, seems like structopt only handles struct as default values

Reply:
But LevelFilter does not implement Default, so we would have to define a wrapper type. We can keep it like that forw now, just need to add the default value to the doc string.

Reply:
right, fixed !